go/vt/vtctl/workflow/server.go

```diff
@@ -1460,13 +1460,21 @@ func (s *Server) moveTablesCreate(ctx context.Context, req *vtctldatapb.MoveTabl
 		return nil, err
 	}
 	sw := &switcher{s: s, ts: ts}
-	lockCtx, targetUnlock, lockErr := sw.lockKeyspace(ctx, ts.TargetKeyspaceName(), "MoveTablesCreate")
+
+	// When creating the workflow, locking the workflow and its target keyspace is sufficient.
+	lockName := fmt.Sprintf("%s/%s", ts.TargetKeyspaceName(), ts.WorkflowName())
+	ctx, workflowUnlock, lockErr := s.ts.LockName(ctx, lockName, "SwitchWrites")
```

Comment:
Can you explain why `contents` is always being passed in as `SwitchWrites` throughout the diff in this file?

Reply:
Good catch! This one should say "MoveTablesCreate". I'll fix that along with any other review related changes.

Reply:
Corrected here: https://github.com/vitessio/vitess/pull/16260/commits/4119a706be18efcf554de8cb9bc7bfdb646bc213